### PR TITLE
Implementation e2e tests

### PR DESCRIPTION
**Pull request by rogowski-piotr**

### CODE REVIEW

File: .github/e2e/endpoints_test.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    params = ""
+    url = f"http://{host}:{port}{endpoint}{params}"
+    payload = {}
+    headers = {}
+
+    # when
+    response = requests.request("GET", url, headers=headers, data=payload)
+
+    # then
+    assert response.status_code == 200
```
Comment (Olczix) [inline .github/e2e/endpoints_test.py:19]: Add assertion failure messages

Example:
assert response.status_code == 200, f"Bad response code {response.status_code}, expected 200"

File: .github/e2e/endpoints_test.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    places_amount_in_response = len(json.loads(response.text))
+    expected_places_amount = 2318
+    assert places_amount_in_response == expected_places_amount
```
Comment (Olczix) [inline .github/e2e/endpoints_test.py:22]: Same here - add assertion message everywhere

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -1,45 +1,58 @@
...
-        run: cd "Spring Boot/microservices/route1-service/" && make test
-        
-      - name: Test route2-service
-        run: cd "Spring Boot/microservices/route2-service/" && make test
-
-  monolith:
+      - name: Build docker environment
+        run: docker-compose -f docker-compose-spring-boot-microservices.yml up --build -d
+
+      - run: pip install -r .github/e2e/requirements.txt
```
Comment (Olczix) [inline .github/workflows/test.yml:16]: Rethink of taking out those 3 steps into some shared variable - there is some code redundancy here

File: .github/e2e/endpoints_test.py (hunk 1 above)
Comment (rogowski-piotr) [reply .github/e2e/endpoints_test.py]: Done :)

File: .github/e2e/endpoints_test.py (hunk 2 above)
Comment (rogowski-piotr) [reply .github/e2e/endpoints_test.py]: Done :)

File: .github/e2e/endpoints_test.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        "coordinate1": "54.367, 18.633",
+        "coordinate2": "50.05, 19.95"
+    })
+    headers = {"Content-Type": "application/json"}
+
+    # when
+    response = requests.request("POST", url, headers=headers, data=payload)
+
+    # then
+    expected_value = 488
```
[the remark this answers is not in the document]
Comment (rogowski-piotr) [reply .github/e2e/endpoints_test.py]: Done :)

File: .github/workflows/test.yml (hunk 3 above)
Comment (rogowski-piotr) [reply .github/workflows/test.yml:20]: No solution found

**Review by Olczix — APPROVED** [review]

**Pull request merged**